README.md

```diff
@@ -201,6 +201,17 @@ The command accepts the following option.
   command will ask you to confirm the creation of the external
   collection. These options specify the answer as a cli option.
 
+```
+beet alt list-tracks [--format=FORMAT] NAME
+```
+
+Lists all tracks that are currently included in the collection.
+
+The `--format` option accepts a [beets path format][path-format] string that is
+used to format each track.
+
+[path-format]: https://beets.readthedocs.io/en/v1.4.9/reference/pathformat.html
```

Comment:
Better change this to the `latest` readthedocs URL.

https://beets.readthedocs.io/en/latest/reference/pathformat.html